modules/appeals_api/spec/docs/v2/legacy_appeals_spec.rb

```diff
@@ -0,0 +1,129 @@
+      operationId 'getLegacyAppeals'
+      security [{ apikey: [] }]
+      consumes 'application/json'
+      produces 'application/json'
+      description = 'Returns eligible Legacy Appeals for a Veteran. A Legacy Appeal is eligible if an SOC/SSOC ' \
+      'has been declared, and if the date of declaration is within the last 60 days. ' \
+      'SOC (Statement of Case) / SSOC (Supplemental Statement of Case)'
+      description description
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_ssn_header]
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_file_number_header]
+
+      response '200', 'Returns eligible legacy appeals for a veteran' do
+        let(:'X-VA-SSN') { '123456789' }
+
+        schema AppealsApi::SwaggerSharedComponents.response_schemas[:legacy_appeals_response_schema]
+
+        before do |example|
+          VCR.use_cassette('caseflow/legacy_appeals_get_by_ssn') do
+            submit_request(example.metadata)
+          end
+        end
+
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+
+        it 'returns a 200 response' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      response '404', 'Veteran record not found' do
+        let(:'X-VA-SSN') { '234840293' }
+
+        schema JSON.parse(
+          File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'errors', '404.json'))
+        )
+
+        before do |example|
+          VCR.use_cassette('caseflow/legacy_appeals_no_veteran_record') do
+            submit_request(example.metadata)
+          end
+        end
+
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+
+        it 'returns a 404 response' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      response '422', 'SSN formatted incorrectly' do
+        let(:'X-VA-SSN') { '12n-~89' }
+
+        schema JSON.parse(
+          File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'errors', 'legacy_appeal_ssn_format.json'))
+        )
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+
+        it 'returns a 422 response' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      # response '422', 'When X-VA-SSN and X-VA-File-Number are missing' do
```

Comment:
Is the plan to flesh this out further or to remove it?

Reply:
I left a comment about this in the engineering chat.